managed/src/main/java/com/yugabyte/yw/models/HealthCheck.java

```diff
@@ -36,15 +60,18 @@
 
   // The Json serialized version of the details. This is used only in read from and writing to the
   // DB.
+  @DbJson
   @Constraints.Required
   @Column(columnDefinition = "TEXT", nullable = false)
-  public String detailsJson;
+  public details detailsJson;
 
   public boolean hasError() {
-    JsonNode details = Json.parse(detailsJson);
-    JsonNode hasErrorField = details.get(FIELD_HAS_ERROR);
```

Comment:
Get rid of static const if not too much of work.

Reply:
Controversial point: Static constants are preferable before string constants used in several places.

Reply:
Been long time gone ... but see changed code. Basically use first class object.